src/packages/src/xroad/ubuntu/generic/control

```diff
@@ -150,7 +150,7 @@ Description: X-Road security server with Estonian settings
 Package: xroad-opmonitor
 Conflicts: xroad-center
 Architecture: all
-Depends: ${misc:Depends}, xroad-base (=${binary:Version}), xroad-confclient (=${binary:Version})
+Depends: ${misc:Depends}, net-tools, iproute2, xroad-base (=${binary:Version}), xroad-confclient (=${binary:Version})
```

Comment:
net-tools, iproute2 are used also by other installation scripts, so putting them in xroad-base -package would probably be better. Instead of netstat (from net-tools), one could use pg_isready.